lib/rex/parser/arguments.rb

```diff
@@ -90,6 +135,16 @@ def arg_required?(opt)
 
       attr_accessor :fmt     # :nodoc:
       attr_accessor :longest # :nodoc:
+
+      private
+
+      def select_value_from_fmt_option(option)
+        fmt_option = fmt.find do |key, value|
+          key = [key] if key.is_a?(String)
```

Comment:
It maybe easier to normalize the data once upfront, instead of handling this edge case individually every time methods try to access this data 😄